### ISSUE TITLE
Loss function with DDP training

### ISSUE THREAD

**Issue by phamdat09** [ultralytics/yolov5#4860]
## ❔Question
Hello, I have a small question. Following the training code, Can you explain why you use the multiplication instead of the division ?
[https://github.com/ultralytics/yolov5/blob/master/train.py#L317](https://github.com/ultralytics/yolov5/blob/master/train.py#L317)

**Comment by glenn-jocher** [comment]
@phamdat09 hi, thank you for your feature suggestion on how to improve YOLOv5 🚀!

The fastest and easiest way to incorporate your ideas into the official codebase is to submit a Pull Request (PR) implementing your idea, and if applicable providing before and after profiling/inference/training results to help us understand the improvement your feature provides. This allows us to directly see the changes in the code and to understand how they affect workflows and performance.

Please see our ✅ [Contributing Guide](https://docs.ultralytics.com/help/contributing/) to get started.
